tutorials/applications/QAOA and QAA to solve a QUBO problem.ipynb

```diff
@@ -40,10 +40,7 @@
     "QUBO has been extensively studied [Glover, et al., 2018](https://arxiv.org/pdf/1811.11538.pdf) and is used to model and solve numerous categories of optimization problems including important instances of network flows, scheduling, max-cut, max-clique, vertex cover and other graph and management science problems, integrating them into a unified modeling framework.\n",
```

Comment:
<p>Perhaps we could add a comment saying that we are making a matrix of the distances between each coordinates ?</p>

<pre class="ql-syntax" spellcheck="false">def evaluate_mapping(new_coords, *args):
&nbsp;&nbsp;&nbsp; """Cost function to minimize. Ideally, the pairwise distances are conserved."""
&nbsp;&nbsp;&nbsp; Q, shape = args
&nbsp;&nbsp;&nbsp; new_coords = np.reshape(new_coords, shape)
    # computing the matrix of the distances between each coordinates
    new_Q = squareform(
  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; DigitalAnalogDevice.interaction_coeff / pdist(new_coords) ** 6
  &nbsp;&nbsp;&nbsp; )
  &nbsp;&nbsp;&nbsp; return np.linalg.norm(new_Q - Q)
</pre><p>Another remark: we use <code>DigitalAnalogDevice</code> in this tutorial, won't it be better to use <code>AnalogDevice</code> ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/731/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>